docs/site/sidebars/lb4_sidebar.yml

```diff
@@ -254,6 +254,10 @@ children:
     url: Serving-static-files.html
     output: 'web, pdf'
 
+  - title: 'Dynamic models, repositories, and controllers'
+  url: Dynamic-models-repositories-controllers.html
```

Comment:
missing indent. I think this .yml needs to have the right indentation in order for it to work.